public/components/query_compare/search_result/result_components/result_grid.tsx

```diff
@@ -27,19 +30,65 @@ interface ResultGridComponentProps {
   comparedDocumentsRank: DocumentRank;
   queryResult: SearchResults;
   resultNumber: number;
+  setQueryError: React.Dispatch<React.SetStateAction<QueryError>>;
```

Comment:
Any reason why setting the `queryError` is required here? Seems like it's being set in `toggleDetails`, but I don't see how it's necessary.

Reply:
Good catch, just removed it.